guides/essences.md

```diff
@@ -13,6 +13,79 @@ Alchemy comes with a lot of predefined essences for the regular needs of a websi
+
+For example. With an `article` element like this:
+
+~~~ yaml
+# config/alchemy/elements.yml
+- name: article
+  contents:
+    - name: headline
+      type: EssenceText
+    - name: color
+      type: EssenceText
+      settings:
+        input_type: color
+    - name: copy
+      type: EssenceRichtext
+~~~
+
+The `_article.html.erb` template generated by `rails alchemy:elements --skip` would look like this:
+
+~~~html
+<%- cache(article) do -%>
+  <%= element_view_for(article) do |el| -%>
+    <div class="headline">
+      <%= el.render :headline %>
+    </div>
+    <div class="color">
+      <%= el.render :color %>
+    </div>
+    <div class="copy">
+      <%= el.render :copy %>
+    </div>
+  <%- end -%>
+<%- end -%>
+
+~~~
+
+Note how `element_view_for` allows you to call `el.render` on essences within the block.
+
+::: tip
+`element_view_for` wraps the block in a `<div>` with a set ID and class by default, but this is [customizeable](https://github.com/AlchemyCMS/alchemy_cms/blob/main/app/helpers/alchemy/elements_block_helper.rb#L137) if you pass in arguments:
+
+`<%= element_view_for(article, tag: :span, id: 'custom_id', class: 'custom_class') do |el| -%>`
+:::
+
+Without using the `element_view_for` helper, you can still access essences:
+
+`article.contents.find_by(name: 'headline')&.ingredient`
+
+But the `el.render` helper takes care of generating the appropriate DOM elements to display the essence based on its type. It is recommended you rely on these helpers unless you are comfortable with the structure of the essence model you are trying to render.
+
+Just like `element_view_for`, you can pass `options` and `html_options` to `el.render`:
+
+`<%= el.render :headline, {#options}, {#html_options} %>`
+
+You can check out the [`ElementViewHelper#render` helper](https://github.com/AlchemyCMS/alchemy_cms/blob/b8cc62493693a070f7457081760708bf28c13e34/app/helpers/alchemy/elements_block_helper.rb#L31) for details.
+
+::: note WARNING
+Options are not universally applied by all element types. For example, `ElementRichText` ignores `html_options` since it's already in HTML format and isn't wrapped in any special div. And you can't prevent the `ElementPicture` from generating an `<img>` tag.
+:::
+
+The `html_option = {}` are:
+* `:tag` - The HTML tag to be used for the wrapping element
+* `:id` - The wrapper tag's DOM ID.
+* `:class` - The wrapper tag's DOM class.
+
+The `options = {}` hash is a lambda used for formatting the child elements' tags. You can pass `false` to not include tags inside the wrapper element. For more, see the source code [here](https://github.com/AlchemyCMS/alchemy_cms/blob/56e95a07446fb6832676acfc1e8c7047d1b80495/app/helpers/alchemy/elements_block_helper.rb#L128) and [here](https://github.com/AlchemyCMS/alchemy_cms/blob/main/app/helpers/alchemy/elements_helper.rb#L193)
```

Comment:
You mean the lambda that can be passed to the `tags_formatter` argument?  We should rephrase this then